### DISCUSSION TITLE
How to disable the search function as it doesn't work with languages other than English

### DISCUSSION THREAD

**Discussion by gitacoco** [GitbookIO/community#786] [1. How to]
Hi Gitbook folks,

Although I have not tested other languages, it seems that it doesn't work with Chinese.
<img width="865" alt="image" src="https://github.com/user-attachments/assets/585146bb-0d0d-49e0-8ab8-fb114dd31887">

If there is no support for other languages, am I able to disable this function? Otherwise, it will be super misleading.

Thank you.

**Comment by JaniceManwiller** [comment]
I would also like to be able to disable the search option for a site, but for a different reason.

One of my sites just provides menus to display information in other sites. There isn't really any content to search in that site, and the search does not extend to the other linked sites.

There has been some confusion because users are expecting the search to search the content in the linked sites, not just the menu site.

**Reply by scazan** [reply]
Hi @JaniceManwiller , Have you checked out our new Site Sections feature? This might be a solve for your issue entirely: https://docs.gitbook.com/changelog#new-and-noteworthy

**Reply by JaniceManwiller** [reply]
No - we do not want to have different products as tabs in a site. Each product is a different site. The doc portal site links to the individual product sites.

We are planning to use the sections, but for different content (main guide, developer's guide) within a single product.

**Comment by scazan** [comment]
> Hi Gitbook folks,
> 
> Although I have not tested other languages, it seems that it doesn't work with Chinese. <img alt="image" width="865" src="https://private-user-images.githubusercontent.com/21225411/383125105-585146bb-0d0d-49e0-8ab8-fb114dd31887.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1MDk1MjYsIm5iZiI6MTczMTUwOTIyNiwicGF0aCI6Ii8yMTIyNTQxMS8zODMxMjUxMDUtNTg1MTQ2YmItMGQwZC00OWUwLThhYjgtZmIxMTRkZDMxODg3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEzVDE0NDcwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0YTYxMjM3NTczNDI5ODkzNTcxYjc0YTdlNzllZTQwMGI5OWUyMDRmZDhmNmYwNDdhMTkyNDhhMmFkYzQzMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vKPMxOQqkcu0JRFR6LjRvCMPlbWlkLAoRBYzXoL4e2Y">
> 
> If there is no support for other languages, am I able to disable this function? Otherwise, it will be super misleading.
> 
> Thank you.

Hi @gitacoco , 

Scott from GitBook here...  We should support Chinese as we have Algolia backing that part of the search (see supported languages here: https://www.algolia.com/doc/guides/managing-results/optimize-search-results/handling-natural-languages-nlp/in-depth/supported-languages/). If you want to share with me your organization id I can take a look and see why you maybe aren't getting the results you hope for.
One other question, is this the search or the Ask AI feature?

Thanks!

**Reply by gitacoco** [reply]
Hi scazan, thank you for your reply. This is for general search, not the Ask AI feature. I tried to find the Org ID according to the guidance from your documentation:
<img width="1168" alt="image" src="https://github.com/user-attachments/assets/90ac9fec-7b1e-4fd6-afb2-201b741bb61f">
However, I couldn't find anything at the same place...
<img width="1007" alt="image" src="https://github.com/user-attachments/assets/ebb9452f-291e-4558-83c5-12860f6a0f04">

**Reply by scazan** [reply]
No worries, I found it from your screenshot. It looks like it is generally working (excuse my lack of language here, I tried searching for "员数量" and came upon my expected search results).

Can you paste the search text and the page on your site that is not showing up as an expected result?

Thanks!
